File: Python/ls.py
def length(list):
  c=0
  for i in list:
    c=c+1
  return c
def search(list,n):
    c=length(list)
    for j in range(c):
        if list[j]==n:
            return 1

a=input("Enter a list : ")
m=input("Enter element to search : ")
z=search(a,m)
if z==1:
    print "The element is in the list"
else:
    print "The number is not in the list"

File: C/arrays/ls.c
#include<stdio.h>
int main()
{
	int a[20],n,i,l,count=0;
	printf("Enter no of elements : ");
	scanf("%d",&n);
	for(int i=0;i<n;i++)
	{
		printf("Enter a value : ");
		scanf("%d",&a[i]);

	}
	printf("Enter element to search : ");
	scanf("%d",&l);
	for(int i=0;i<n;i++)
	{
		if(a[i]==l)
		{
			count+=1;
		}

	}
	printf("The element is found and its count is %d\n",count);

	
}